wiki/Medals/Unlock_requirements/Skill/id.md

```diff
@@ -0,0 +1,32 @@
+# Persyaratan untuk mendapatkan medali Skill
+
+| Nama medali | Persyaratan |
+| :-- | :-- |
+| 500 Combo | Capai 500 combo pada sebuah beatmap dengan mode permainan apapun. |
```

Comment:
`combo` to `kombo`

Reply:
I'm keeping the `combo` and `full combo` to sync with [\[ID\] Combo](https://github.com/ppy/osu-wiki/blob/master/wiki/Glossary/Combo_(score_multiplier)/id.md) and [\[ID\] Full Combo](https://github.com/ppy/osu-wiki/blob/master/wiki/Full_combo/id.md) articles